### PR TITLE
[WWW-92] Pipelines subpage

### PR DESCRIPTION
**Pull request by klijakub**
Hello, 
- new pipelines subpage ( /pipelines routing)
- added new assets from the approved design 
- features section

Jira Item: [WWW-92]

[WWW-92]: https://gruntwork.atlassian.net/browse/WWW-92

### CODE REVIEW

**Comment by eak12913** [comment]
@klijakub I'm not sure if this is just something on my end - but the `terraform plan` and `terraform apply` seem like they are much larger than the surrounding text. See the screenshot:

image

I think @josh-padnick will be the primary person to review this - but just stood out to me as a little odd. You may want to make it a bit smaller.

**Comment by josh-padnick** [comment]
Thanks for submitting this PR! I just submitted a commit with the final set of changes, and this is now ready to merge.

@eak12913 @oredavids Could one of you approve and merge this as I'm now a contributor?

@klijakub There is still one additional change to make. We actually got some extra feedback on the main image. Could you make the following updates to that image?
   - Remove the Lambda function from the diagram
   - Change "CI Server" to "Your CI Server"
   - Add another label to the CI Server called "Pipelines CLI" with the Grunty logo
   - Add another label "Your Terraform Code" to the Terraform logo

**Review by josh-padnick — DISMISSED** [review]

**Comment by klijakub** [comment]
> Thanks for submitting this PR! I just submitted a commit with the final set of changes, and this is now ready to merge.
> 
> @eak12913 @oredavids Could one of you approve and merge this as I'm now a contributor?
> 
> @klijakub There is still one additional change to make. We actually got some extra feedback on the main image. Could you make the following updates to that image?
> 
> * Remove the Lambda function from the diagram
> * Change "CI Server" to "Your CI Server"
> * Add another label to the CI Server called "Pipelines CLI" with the Grunty logo
> * Add another label "Your Terraform Code" to the Terraform logo
@josh-padnick 
Thanks for the review! :) Sure I will let know our design team and we will get back to you with the new diagram

File: _data/pipelines-features.yml
Comment (brikis98) [inline _data/pipelines-features.yml:1]:
```suggestion
- title: Set up a Terraform pipeline based on best practices
```

File: _data/pipelines-features.yml
Comment (brikis98) [inline _data/pipelines-features.yml:11]:
```suggestion
  description: Run Gruntwork pipelines on any CI tool you want, including both managed and self-hosted varieties. There's no need to set up a new tool, or share sensitive cloud credentials with another third party.
```

File: pages/pipelines/_docs.html
Comment (brikis98) [inline pages/pipelines/_docs.html:7]:
```suggestion
      code for the full architecture plus step-by-step instructions on how to set up Gruntwork Pipelines in your environment.
```

File: pages/pipelines/_sub-hero.html
Comment (brikis98) [inline pages/pipelines/_sub-hero.html:6]:
```suggestion
      to use your preferred CI tool to set up an end-to-end pipeline for infrastructure 
```

File: _data/pipelines-features.yml
Comment (brikis98) [inline _data/pipelines-features.yml:7]:
```suggestion
  description: It's all too common to give your CI server admin-level permissions needed to launch arbitrary infrastructure, a dangerous security posture. Gruntwork Pipelines enables a secure infrastructure pipeline by granting the sensitive permissions solely to a Fargate Task that exposes a limited, locked down interface and only allows running pre-defined commands such as <code>terraform apply</code> in pre-defined git repos/branches/folders that require peer review to change.
```

**Review by brikis98 — COMMENTED** [review]
Just added a few trivial grammar NITs

**Comment by klijakub** [comment]
@brikis98 fixed! :)

@josh-padnick the new diagram is ready and committed: 

image

**Comment by eak12913** [comment]
@klijakub Could you please update your branch and then I can approve it? Otherwise I believe my approval may be dismissed if you push additional commits after I approve.

Thanks and then I'll be able to approve and merge!

**Comment by klijakub** [comment]
> @klijakub Could you please update your branch and then I can approve it? Otherwise I believe my approval may be dismissed if you push additional commits after I approve.
> 
> Thanks and then I'll be able to approve and merge!

Sorry, didn't notice your comment. Sure, I will do it asap

**Comment by josh-padnick** [comment]
@klijakub Thanks for the fast turnaround! Ok, just one more minor change.  Can you modify "Pipelines CLI" to "Gruntwork Pipelines CLI"? Otherwise, everything looks great.

**Comment by josh-padnick** [comment]
@eak12913 Could you give me another approval? Had to resolve merge conflicts before I could merge.

**Review by eak12913 — DISMISSED** [review]
Rubber stamping. Thanks!

**Comment by josh-padnick** [comment]
@oredavids Sorry, looks like I need yet one more non-contributor approval here...not sure why?

**Comment by klijakub** [comment]
> @klijakub Thanks for the fast turnaround! Ok, just one more minor change. Can you modify "Pipelines CLI" to "Gruntwork Pipelines CLI"? Otherwise, everything looks great.

Done ;)

**Comment by josh-padnick** [comment]
Thanks for the fast turnaround! This is ready to merge once we can get one more non-contributor approval.

**Comment by josh-padnick** [comment]
@eak12913 Could you (a) resolve this merge conflict, and (b) coordinate with another (non-contributor) Grunt to get this deployed?

**Comment by eak12913** [comment]
I will take a look!

**Comment by eak12913** [comment]
@josh-padnick @klijakub: I just cloned Altology's fork of this repo and tried to resolve the conflict (which I was able to do) but I can't actually push to this repo.

```
Pushing to https://github.com/Altalogy/gruntwork-io.github.io.git
remote: Permission to Altalogy/gruntwork-io.github.io.git denied to eak12913.
fatal: unable to access 'https://github.com/Altalogy/gruntwork-io.github.io.git/': The requested URL returned error: 403
```

The fix is simple - for `prism.css` just accept what's coming from Gruntwork. I'm not even sure if it needed to change in this PR as we don't appear to be using prism to render any code on the Pipelines page.

**Comment by klijakub** [comment]
> @josh-padnick @klijakub: I just cloned Altology's fork of this repo and tried to resolve the conflict (which I was able to do) but I can't actually push to this repo.
> 
> ```
> Pushing to https://github.com/Altalogy/gruntwork-io.github.io.git
> remote: Permission to Altalogy/gruntwork-io.github.io.git denied to eak12913.
> fatal: unable to access 'https://github.com/Altalogy/gruntwork-io.github.io.git/': The requested URL returned error: 403
> ```
> 
> The fix is simple - for `prism.css` just accept what's coming from Gruntwork. I'm not even sure if it needed to change in this PR as we don't appear to be using prism to render any code on the Pipelines page.

Oh sorry, @eak12913 , you weren't a member of the forked repo, I sent you an invite, please let me know if you're able now?

**Comment by eak12913** [comment]
Thanks @klijakub! I'm in and was able to push. Now - I don't think that I will be able approve this - so we may need someone else to come and help us. @josh-padnick @oredavids ?

**Review by eak12913 — APPROVED** [review]
I checked all the pages and saw no weird prism CSS issues (checked Guides, LZ page and the pipelines page)

**Comment by josh-padnick** [comment]
Ugh, I'm a contributor now. @oredavids can you approve?

**Review by oredavids — APPROVED** [review]
Rubber stamp.

**Pull request merged**
